e2e/runAllTests.js

```diff
@@ -0,0 +1,55 @@
+import { execSync } from 'node:child_process';
+import fs from 'node:fs/promises';
+import path from 'node:path';
+import process from 'node:process';
+
+const TEST_COMMAND = 'npm run lint';
+
+const getRoot = () => {
+  return execSync('git rev-parse --show-toplevel', {
+    encoding: 'utf-8',
+  }).trim();
+};
+
+const executeAllE2eTests = async () => {
+  const e2eDir = path.resolve(getRoot(), './e2e');
+  const failedTests = [];
+
+  // Get all directories in the e2e dir
+  const testDirs = (await fs.readdir(e2eDir, { withFileTypes: true }))
+    .filter((dirEnt) => dirEnt.isDirectory())
+    .map((dirEnt) => path.join(dirEnt.parentPath, dirEnt.name));
+
+  for (const testDir of testDirs) {
+    const dirName = path.basename(testDir);
+    console.log(`🧪 Executing test: ${dirName}`);
+
+    // Run install and the test command
+    execSync('npm install', {
+      cwd: testDir,
+      stdio: ['ignore', 'ignore', 'pipe'],
+    });
+    try {
+      execSync(TEST_COMMAND, {
+        cwd: testDir,
+        stdio: ['ignore', 'ignore', 'pipe'],
+      });
+      console.log(`✅ Test passed\n`);
+    } catch (error) {
+      console.log(`❌ Test failed`);
+      console.error(`${error}\n`);
```

Comment:
Will this print the actual lint violations to help with debugging? I don't see the actual errors in the screenshot. This means someone would have to rerun locally which is not ideal.

Reply:
That's a good point.  It's only outputting which script command failed. 
<img width="279" height="32" alt="image" src="https://github.com/user-attachments/assets/4a0fe6f6-7d58-4bae-98c1-b96a78ffd3c5" />

But not what caused the command to fail.  I'll improve that

Reply:
What do you think about this?  

<img width="771" height="523" alt="image" src="https://github.com/user-attachments/assets/7d6d20d2-7520-4048-9cc2-e2f86cd0686d" />

Reply:
Seems fine.